File: TwinPrime.c
#include <stdio.h>
#include <math.h>

int isPrime(int n) {
    if (n < 2) return 0;

    for (int i = 2; i <= sqrt(n); i++) {
        if (n % i == 0)
            return 0;
    }
    return 1;
}

int main() {
    int start, end;

    printf("Enter start of range: ");
    scanf("%d", &start);

    printf("Enter end of range: ");
    scanf("%d", &end);

    printf("Twin prime pairs:\n");

    for (int i = start; i <= end - 2; i++) {
        if (isPrime(i) && isPrime(i + 2)) {
            printf("(%d, %d)\n", i, i + 2);
        }
    }

    return 0;
}

File: TwinPrime.py
import math

def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(math.sqrt(n)) + 1):
        if n % i == 0:
            return False
    return True

start = int(input("Enter start of range: "))
end = int(input("Enter end of range: "))

print("Twin prime pairs:")

for i in range(start, end - 1):
    if is_prime(i) and is_prime(i + 2):
        print(f"({i}, {i+2})")
